packages/braintree-integration/src/braintree-local-payment-methods/braintree-local-methods-payment-strategy.ts

```diff
@@ -215,4 +263,27 @@ export default class BraintreeLocalMethodsPaymentStrategy implements PaymentStra
             onError(error);
         }
     }
+
+    /**
+     *
+     * Utils
+     *
+     * */
+    private isNonInstantPaymentMethod(methodId: string): boolean {
+        return methodId.toUpperCase() in NonInstantLocalPaymentMethods;
+    }
+
+    private isBraintreeTrustlyRedirectError(error: unknown) {
```

Comment:
I recommend to update this method with abstract naming, so we can use it for future implementations (for example if we will implement OXXO for Braintree or another payment method that might have such redirect flow)

```suggestion
    private isBraintreeRedirectError(error: unknown) {
```